templates/frontend/pages/announcements.tpl

```diff
@@ -23,15 +23,13 @@
 
 			{$announcementsIntroduction|strip_unsafe_html}
 
-			<ul class="announcements__toc">
-				{foreach from=$announcements item=announcement}
-					<li>
-						{include file="frontend/objects/announcement_summary.tpl"}
-					</li>
-				{/foreach}
-			</ul>
+			{foreach from=$announcements item=announcement}
+				<article>
+					{include file="frontend/objects/announcement_summary.tpl"}
+				</article>
```

Comment:
Yeah so whatever you prefer, but here is where I was saying in most cases that root element is part of the template.

Reply:
Will change it now for better consistency!